src/handlers/genreById.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import getAll from '../models/Genre/getAll';
+import { GenreResponse } from '../types/apiResponse';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+	const notFoundResponse = {
+		statusCode: 404,
+		body: JSON.stringify({ message: 'Not Found' }),
+	};
```

Comment:
Maybe make this a global instead of a file-specific thing? For serverless, I typically make a response generator function that I share. It let's me standardize 404, 500 messages and makes sure the right structure is always returned.